gateway-service/src/test/java/org/zowe/apiml/gateway/security/service/schema/ZoweJwtSchemeTest.java

```diff
@@ -98,7 +98,7 @@ void whenValidJWTAuthSource_thenUpdateZuulHeaderWithJWToken() {
             when(authSourceService.parse(authSource)).thenReturn(new ParsedTokenAuthSource("user", new Date(), new Date(), Origin.ZOSMF));
             command = scheme.createCommand(null, authSource);
             command.apply(null);
-            verify(requestContext, times(1)).addZuulRequestHeader(any(), any());
+            verify(requestContext, times(3)).addZuulRequestHeader(any(), any());
```

Comment:
Why is this test now ran three times?

Reply:
now, its running 2 more times to set authorization header with JWT